src/ompl/geometric/planners/rrt/RRTX.h

```diff
@@ -0,0 +1,470 @@
+
+            /** \brief A constant for k-nearest rewiring calculations */
+            double                                         k_rrg_;
+            /** \brief A constant for r-disc rewiring calculations */
+            double                                         r_rrg_;
+
+            /** \brief Objective we're optimizing */
+            base::OptimizationObjectivePtr                 opt_;
+
+            /** \brief The most recent goal motion.  Used for PlannerData computation */
+            Motion                                         *lastGoalMotion_;
+
+            /** \brief A list of states in the tree that satisfy the goal condition */
+            std::vector<Motion *>                           goalMotions_;
+
+            /** \brief Best cost found so far by algorithm */
+            base::Cost                                     bestCost_;
+
+            /** \brief Number of iterations the algorithm performed */
+            unsigned int                                   iterations_;
+
+            /** \brief Comparator of motions, used to order the queue */
+            MotionCompare                                   mc_;
+
+            /** \brief Queue to order the nodes to update */
+            BinaryHeap<Motion *, MotionCompare>             q_;
+
+            /** \brief Threshold for the propagation of information */
+            base::Cost                                      epsilonCost_;
+
+            /** \brief Whether or not to propagate the cost to children if the update is less than epsilon */
+            bool                                            updateChildren_;
+
+            /** \brief Current value of the radius used for the neighbors */
+            double                                          rrg_r_;
+
+            /** \brief Current value of the number of neighbors used */
+            unsigned int                                    rrg_k_;
+
+            /** \brief Variant used for rejection sampling */
+            int                                             variant_;
+
+            /** \brief Alpha parameter, scaling the rejection sampling tests */
+            double                                          alpha_;
+
+            /** \brief Option to use informed sampling */
+            bool                                            useInformedSampling_;
+
+            /** \brief The status of the sample rejection parameter. */
+            bool                                            useRejectionSampling_;
+
+            /** \brief The number of attempts to make at informed sampling */
+            unsigned int                                    numSampleAttempts_;
+
+            ///////////////////////////////////////
+            // Planner progress property functions
+            std::string numIterationsProperty() const
+            {
+                return boost::lexical_cast<std::string>(numIterations());
+            }
```

Comment:
We're trying to use the standard C++ library when possible. In this case, you can use [std::to_string](http://en.cppreference.com/w/cpp/string/basic_string/to_string).